packages/components/src/FilterDatePicker/subcomponents/FilterDatePickerField/FilterDatePickerField.tsx

```diff
@@ -78,7 +83,7 @@ export const FilterDatePickerField = ({
   classNameOverride,
   ...restProps
 }: FilterDatePickerFieldProps): JSX.Element => {
-  const [id] = useState<string>(propsId || v4())
+  const [id] = useState<string>(propsId || useId())
```

Comment:
Should this be `const id = propsId ?? useId()`

Reply:
Nope apparently it should be this (but yes thanks good catch)!
```
const reactId = useId()
const id = propsId ?? reactId
```